docs/source/examples/notebooks/getting_started/tutorial-8-solver-options.ipynb

```diff
@@ -89,26 +77,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 11,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Safe mode solve time: 137.215 ms\n",
-      "Fast mode solve time: 92.051 ms\n"
+      "Safe mode solve time: 16.418 ms\n",
+      "Fast mode solve time: 83.043 ms\n"
```

Comment:
This does not seem to match what is above anymore. Since the old one was about the safe/fast modes of casadi